source/plugin/permissions.rst

```diff
@@ -370,3 +370,32 @@ The ``ContextCalculator`` can be registered via:
 .. code-block:: java
     
     permissionService.registerContextCalculator(contextCalculator);
+
+For Forge Mods
+========
+
+If you are the author of a Forge mod and are not using the new Forge PermissionsAPI but are doing OP checks then you are already on the right path for Sponge to pick up permissions.
+
+The simplest way to create a Sponge permission in a Forge mod without soft-depending on SpongeAPI is to use the method provided by Vanilla Minecraft code in ``ICommandSender``, namely ``ICommandSender.canCommandSenderUseCommand(int permLevel, String commandName)``. The String passed into that method has no use at all in a Vanilla Forge environment but when Sponge Forge is added it automatically takes that String and converts it into a working permission.
```

Comment:
SpongeForge should be one word. Also, line lengths should be shorter (120 characters or less) to meet our usual guidelines.

Reply:
I had done a quick search for the word Forge earlier and noticed that Sponge Forge came up in the majority of the results, I thought it was odd and I will note it in the OCD PR